lib/endpoints/class-wp-rest-terms-controller.php

```diff
@@ -528,6 +489,52 @@ public function delete_item( $request ) {
 	}
 
 	/**
+	 * Prepare a single term for create or update
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 * @return object $prepared_term Term object.
+	 */
+	public function prepare_item_for_database( $request ) {
+		$prepared_term = new stdClass;
+		$prepared_term->args = array();
+
+		if ( isset( $request['name'] ) ) {
+			$prepared_term->name = $request['name'];
+			$prepared_term->args['name'] = $request['name'];
```

Comment:
I see what you mean, but if I return a single term object then would that still work with the `wp_insert_term` function?  I see that you need to supply both the name and the args separately which is why I nested them that way.

https://github.com/WP-API/WP-API/blob/develop/lib/endpoints/class-wp-rest-terms-controller.php#L360

Do you think a better way would be to return a WP_Term object and then supply the entire object as the args and then just use the name attribute for the first parameter of `wp_insert_term`?

Maybe like this `$term = wp_insert_term( $prepared_term->name, $this->taxonomy, $prepared_term );`